### DISCUSSION TITLE
Create a group of users

### DISCUSSION THREAD

**Discussion by FJBDev** [mytourbook/mytourbook#1218] [Ideas]
As I just created [this feature request](https://github.com/mytourbook/mytourbook/issues/1217), I was thinking I would have liked to ask active users their feedback, critics, suggestions etc....
Because I realized that a lot of you are very active in using MyTourbook, reporting bugs, testing features, testing beta versions....etc

Would it be useful to create a group/team (see Github doc [here](https://docs.github.com/en/organizations/organizing-members-into-teams/creating-a-team)) so that we can "mention" this group whenever needed ?

Thoughts ?

@wolfgang-ch @telemaxx @stvzldn @mff73 @Marty56 @ovahead @GurVJ @bern6 @Thompson-ongithub
Sorry if I forgot anyone and sorry if I "mentioned" you but you are not interested (let me know and I'll remove you) !

**Comment by wolfgang-ch** [comment]
I created a group @mytourbook/active-users 

This feature is new for me so I don't know if you can create a request to be a member of this group but I could invite you to be a member.

![image](https://github.com/mytourbook/mytourbook/assets/1283445/aaeda2fd-1621-4778-8980-1b01daa4d83e)

![image](https://github.com/mytourbook/mytourbook/assets/1283445/5d87ff19-a6be-4f6b-842a-e6ab0be77138)

**Reply by telemaxx** [reply]
I have just requested to become a membership. 

hopefully @wolfgang-ch accept it 😉

**Reply by wolfgang-ch** [reply]
![image](https://github.com/mytourbook/mytourbook/assets/1283445/ca7cbfab-0afe-41e0-adba-219f49f0d568)

**Reply by wolfgang-ch** [reply]
I've a new attribute 😄 

![image](https://github.com/mytourbook/mytourbook/assets/1283445/755a9070-3c00-4982-bebd-7e1e3883e2b3)

**Comment by FJBDev** [comment]
> I created a group @mytourbook/active-users


Thanks, I was able to use it in [this feature request](https://github.com/mytourbook/mytourbook/issues/1217) 

> This feature is new for me so I don't know if you can create a request to be a member of this group but I could invite you to be a member.
> 

This is also new to me but I have requested access

**Comment by GurVJ** [comment]
Sorry but I can't find anywhere to request access (I am not familiar with github).

**Reply by FJBDev** [reply]
@GurVJ 
Go to [this link](https://github.com/orgs/mytourbook/teams/active-users) and you should see a "Request access" button. If not, attach a screenshot so we can see what you see

**Reply by GurVJ** [reply]
That is what I've tried but I get a 404 error page.
![image](https://github.com/mytourbook/mytourbook/assets/64196685/1a01be5a-13db-4818-923a-8acddd5da29f)

**Reply by telemaxx** [reply]
@GurVJ: I think you have to login into github first

**Comment by Thompson-ongithub** [comment]
> @GurVJ: I think you have to login into github first

being logged in doesn't change a thing (at least not for me). I can see the same 404 page as GurVJ

**Reply by stvzldn** [reply]
Same issue with the 404 for me - I am logged in.

**Comment by wolfgang-ch** [comment]
I've created invites for @GurVJ @stvzldn @Thompson-ongithub

**Comment by mff73** [comment]
Hi, sorry for being late to the party.
Same issue 404 error on my side, even logged in, to join this group 😊

**Reply by wolfgang-ch** [reply]
These are the current party members

![image](https://github.com/mytourbook/mytourbook/assets/1283445/72b482dc-57b2-4ce9-81b0-18d77f632d3f)
